### PR TITLE
chore: contribution guidelines

### PR DESCRIPTION
**Pull request by yohanelly95**
resolves 
- #64

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+# Contributing to Razor Network Datasources
+If you wish to propose a new price feed to be added to the datasources used by the protocol, please follow the steps below:
+
+1. Start by forking the repository and creating your branch from `master`.
+2. Add the new job to `mainnet/jobs.json` and update `mainnet/collections.json` as needed with the correct jobIDs. (an example for `testnet` is given below)
+    - For example, adding a new `ETH/USD` job to an existing collection for testnet:
+```json
+[
+  {
+    "weight": 1,
```
Comment (SkandaBhat) [inline CONTRIBUTING.md:11]: Please explain what each attribute is (weight, power, selectorType, name, selector, URL, etc) and illustrate why its important.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    "url": "https://www.example.com/"
+  }
+]
+```
+Now to include this new job into an existing collection in `testnet/collections.json` we need to add the serialized jobID to the desired collection. In this case we are adding `ETH/USD` job to the existing `ethCollectionMedian` collection:
+
+```json
+[
+  {
+    "jobIDs": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10], // 10 is added here which is the jobId of the new  ethusd_test job
```
Comment (SkandaBhat) [inline CONTRIBUTING.md:34]: Please explain the attributes

**Review by SkandaBhat — CHANGES_REQUESTED** [review]
Please explain the attributes of jobs and collections clearly.

File: CONTRIBUTING.md (hunk 1 above)
Comment (yohanelly95) [reply CONTRIBUTING.md:11]: noted

File: CONTRIBUTING.md (hunk 2 above)
Comment (yohanelly95) [reply CONTRIBUTING.md:34]: noted

File: CONTRIBUTING.md (hunk 1 above)
Comment (yohanelly95) [reply CONTRIBUTING.md:16]: resolved here [6b4dd2c](https://github.com/razor-network/datasources/pull/55/commits/6b4dd2c39021acbfdeaa19e752379cd5d63f9345)

File: CONTRIBUTING.md (hunk 2 above)
Comment (yohanelly95) [reply CONTRIBUTING.md:39]: resolved here [6b4dd2c](https://github.com/razor-network/datasources/pull/55/commits/6b4dd2c39021acbfdeaa19e752379cd5d63f9345)

**Review by SkandaBhat — APPROVED** [review]

**Review by SkandaBhat — APPROVED** [review]

**Pull request merged**